.pylintrc

```diff
@@ -1,6 +1,6 @@
 [MESSAGES CONTROL]
 
-disable=broad-except,len-as-condition,logging-fstring-interpolation,logging-not-lazy
+disable=broad-except,len-as-condition,logging-fstring-interpolation,logging-not-lazy,unexpected-line-ending-format
```

Comment:
This is a fix for VS code messing with line endings, right?  I'm wondering because Vim can't handle mixed line endings at all nicely (which made CompLib's code base unnecessarily hard to work with :laughing:).

Reply:
This is `unexpected-line-ending-format` not `mixed-line-endings` which would error if both CRLF and LF used in a single file.
I've actually enabled this warning again on linix in `run-checks.py`.

The behaviour is reasonable git defaults on Windows to convert to CRLF on checkout and to LF on commit (e.g. if you create a new file it's likely to be CRLF.) VS Code just shows all the pylint warnings on each line.

Reply:
Cool - that makes sense.